kubernetes/loculus/values.yaml

```diff
@@ -81,7 +110,7 @@ defaultOrganismConfig: &defaultOrganismConfig
             sampleCollectionDate: sampleCollectionDate
           args:
             order: [geoLocCountry, accession_version, sampleCollectionDate]
-            type: [string, string, date]
+            type: [string, string, string]
```

Comment:
I failed to apply this change here to Pathoplexus values.yaml - this caused the bug: https://github.com/pathoplexus/pathoplexus/issues/533